src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/keycloak-client/outputs.tf

```diff
@@ -12,3 +12,10 @@ output "config" {
     callback_urls      = var.callback-url-paths
   }
 }
+
+output "client_role_ids" {
```

Comment:
I need to get the role id for allow-app-sharing-role so I can assign it to service-account-jhub-apps user.